### ISSUE TITLE
Horticulturalist Web UI drops the upgrading throbber once api has done its part instead of waiting until a new ddoc is ready

### ISSUE THREAD

**Issue by SCdF** [medic/medic-webapp#3967]
When clicking upgrade we show an upgrade message and a loading throbber, and fire a request off to api to stage the new release for horticulturalist to pick up.

Currently once api returns we drop the throbber and go back to the upgrade page, with no indication of what's going on. However, the upgrade isn't complete until horti picks it up, kills api / sentinel, deploys, and starts api / sentinel again. At which time, we'll automatically reconnect and get the "there is a new version of this software" model.

We should change the code so the message and throbber never get dropped, so that they will stick around until the new version modal appears.

**Comment by ngaruko** [comment]
LGTM. Except (and I might have nothing to do with the issue at hand, but worth mentioning and will create a new issue), pressing the `Cancel` button on the 'Update Available` modal freezes horti page (same behavior if one clicks outside the modal) and once one refreshes the page, then the new version is loaded anyway.  
![image](https://user-images.githubusercontent.com/6979995/36700065-7ece4546-1bb3-11e8-8a6d-d99d27b0d981.png)

**Comment by garethbowen** [comment]
@ngaruko What do you mean by freezes?

**Comment by ngaruko** [comment]
@garethbowen 
Maybe not freezing as such, but the upgrading throbber spins forever and the page only reloads if ones moves to another page or refresh otherwise. 
![image](https://user-images.githubusercontent.com/6979995/36701059-ba7280f0-1bb6-11e8-91de-f3405f9ae1d1.png)

**Comment by garethbowen** [comment]
Yes that's because of this change. I think that's probably ok for now. What do you think?

**Comment by ngaruko** [comment]
Not sure if the update is reversible at this stage (actually the same behavior shows when using the 'Dashboard`, except for the spinner. I mean, the update still happens even on pressing `Cancel` or ignoring the pop up. So, I think, if the update is not reversible at this stage, the best way would be to just show an 'information` pop-up. I have also realised that this button is also ignored for other changes, such as changing permissions...

**Comment by garethbowen** [comment]
The update is never reversible which is why we have lots of warnings before you hit "upgrade". If it's not clear, raise issues.

**Comment by ngaruko** [comment]
I will send this back to dev to fix just the spinner issue. Right @garethbowen @dianabarsan ? No need for another issue.

**Comment by garethbowen** [comment]
Due to the way our app is built it will be quite difficult to stop the spinner once the upgrade is finished - we just don't know when it happens. Please raise a new issue.
